public/app/features/dashboard/dashgrid/DashboardGrid.tsx

```diff
@@ -285,7 +362,7 @@ interface GrafanaGridItemProps extends React.HTMLAttributes<HTMLDivElement> {
   isViewing: boolean;
   windowHeight: number;
   windowWidth: number;
-  children: any;
+  children: any; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
Why is this change here?

Reply:
This kept failing the pre-hook linting. As far as I understand this was committed before that lint check was activated or someone was able to bypass the linter when committing this line.